plugin/plugin.go

```diff
@@ -500,8 +520,35 @@ func (p *plugin) generateFloatValidator(variableName string, ccTypeName string,
 	}
 }
 
+// getUUIDRegex returns a regex to validate that a string is in UUID
+// format. The version parameter specified the UUID version. If version is 0,
+// the returned regex is valid for any UUID version
+func getUUIDRegex(version *int32) (string, error) {
+	if version == nil {
+		return "", nil
+	}
+
+	if *version < 0 || *version > 5 {
+		return "", fmt.Errorf("UUID version should be between 0-5, Got %d", *version)
+	}
+
+	if *version == 0 {
+		return fmt.Sprintf(uuidPattern, "1-5"), nil
+	}
```

Comment:
Nit: same here to move to an `if ... else if` chain to state the disjunction of cases.

Reply:
I think the current variant was easier to follow and read but I changed it to your suggestion. :-)